src/components/App.tsx

```diff
@@ -47,10 +48,13 @@ class App extends TsxComponent<AppProps> {
   @Prop({ required: true }) defaultLocale!: AppProps["defaultLocale"];
   @Prop({ required: true }) handleRouteChange!: AppProps["handleRouteChange"];
   @Prop() fsTppVersion: AppProps["fsTppVersion"];
+  @Prop({ default: false }) customSnapHooks!: AppProps["customSnapHooks"];
```

Comment:
You wrote in your comment in JIRA: `this defaults could be opt-out via fsxa.customSnapHooks`
I see this reflected in the code, but I am confused from where and how this flag is meant to be passed to this App component. Specifically, suppose I want to start the fsxa-pwa with `customSnapHooks: true`. Where do I need to set this flag and using which mechanism will it be passed through to the component?

I tried looking at how other props are passed through and currently it seems to me that we need an additional PR in the fsxa-nuxt-module that will add this passthrough logic.

Reply:
@christian-breidler  will check this

Reply:
I added this https://github.com/e-Spirit/fsxa-nuxt-module/pull/92.